ivadomed/inference.py

```diff
@@ -240,19 +242,19 @@ def process_transformations(context: dict, fname_roi: str, fname_prior: str, met
     Returns:
         dict: metadata.
     """
-    if fname_roi is None and 'ROICrop' in context["transformation"].keys():
+    if fname_roi is None and TransformationKW.ROICROP in context[ConfigKW.TRANSFORMATION].keys():
         logger.warning(
             "fname_roi has not been specified, then a cropping around the center of the image is "
             "performed instead of a cropping around a Region of Interest.")
 
-        context["transformation"] = dict((key, value) if key != 'ROICrop'
-                                         else ('CenterCrop', value)
-                                         for (key, value) in context["transformation"].items())
+        context[ConfigKW.TRANSFORMATION] = dict((key, value) if key != TransformationKW.ROICROP
+                                         else (TransformationKW.CENTERCROP, value)
+                                         for (key, value) in context[ConfigKW.TRANSFORMATION].items())
```

Comment:
This is rather ... hard to read. I recommend breaking it into loops and properly expand it out.